core/pkg/hostsensorutils/log_coupling_test.go

```diff
@@ -0,0 +1,98 @@
+package hostsensorutils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLogsMap_Update(t *testing.T) {
+	tests := []struct {
```

Comment:
I think we should run this test in parallel.

Reply:
Ok, I'm going to add this